src/figuresClasses.ts

```diff
@@ -1,11 +1,99 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  color: 'red' | 'green' | 'blue';
+
+  shape: 'triangle';
```

Comment:
The `shape` property in the `Triangle` class is declared as a string literal type `'triangle'`. However, it should be declared as `shape: 'triangle' = 'triangle';` to correctly initialize it with the value `'triangle'`.